OOP/PaymentMenu

```diff
@@ -0,0 +1,42 @@
+public class PaymentMenu extends Menu{
+
+	public PaymentMenu() {
+		Action action=new CCAction();
```

Comment:
Since the only use of 'action' variable and likewise 'action1' and 'action2' variable is to pass it into a constructor, the instance could be directly used. E.g. 
    addMenuItem(new ActionableMenuItem("enter 1 for cc", new CCAction()));

This way u can save local variables.
BUT this is not hard-n-fast. If u feel that if this approach gives lesser readability then avoid this.